charts/spin-operator/templates/mutating-webhook-configuration.yaml

```diff
@@ -13,12 +13,13 @@ webhooks:
     service:
       name: '{{ include "spin-operator.fullname" . }}-webhook-service'
       namespace: '{{ .Release.Namespace }}'
+      # TODO @asteurer: Should this filepath be updated?
```

Comment:
Yes, the `spinoperator-dev` is the domain of the resource with `.` replaced by `-` to be a valid url path.